docs/TargetMaintenance/CreatingAUnifiedTarget.md

```diff
@@ -55,7 +55,7 @@ These firmware types can be created by building locally and specifying the MCU t
 
 - make sure to enable 'Full chip erase' before flashing, or reset to default with `defaults` in CLI.
 
-- verify that your board is properly reset to defaults: The output of a `diff hardware` in CLI should show `board_name`, `manufacturer_id`, and lines starting with a `#`.
+- verify that your board is properly reset to defaults.
 
 ### 2.3 Resource Mappings
```

Comment:
this section should also mention 
1) configuring the gyro, acc, baro, mag, bus type and bus instance and CS pins. 
2) any OSD bus/instance and SPI CS pin settings.
3) any other always-present board hardware, like VTX pit switches, Camera Switchers, PINIO settings.